components/index/Birthday.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="c-birthday" id="birthday">
+    <div class="c-birthday__content">
+      <div class="c-birthday__photo">
+        <img src="~/static/images/index/birthday.png" />
+      </div>
+      <div class="c-birthday__text">
+        <h2 class="c-birthday__header">{{ $t('Celebrating ten years of CoderDojo') }}</h2>
+        <p>{{ $t('This year we’re celebrating ten years of CoderDojo. Discover how CoderDojo started and how you can get involved in the exciting activities we have planned!') }}</p>
+        <div class="c-birthday__text-link">
+          <a href="/">{{ $t('Click link') }}</a>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+  @import '~/assets/_variables.scss';
+
+  .c-birthday {
+    padding: 40px 0;
+
+    &__content {
+      background: $foam;
+      max-width: $content-width;
+      margin: 40px auto;
+      display: flex;
+    }
+
+    &__photo {
+      display: flex;
+      flex: 1;
+      margin: 20px;
+
```

Comment:
Image is stretched to a square in the preview.  Maybe adding `align-self: center;` on .c-birthday__photo will fix - behaves in browser and doesn't affect in responsive view.